app/src/main/res/layout/fragment_add_product_ingredients.xml

```diff
@@ -336,6 +336,7 @@
     <Button
         android:id="@+id/btn_next"
         style="@style/ButtonFlat"
+        android:layout_height="40dp"
```

Comment:
Is this part of this PR?

Reply:
Did this to make the toast visible

Reply:
What do you mean with "visible"? The toast should display over everything in the screen. Could you provide a screenshot of the issue?

Reply:
Yes I am sending

Reply:
@VaiTon I think there was some glitch earlier, I can see the toast now. 
I have made the change.

Reply:
Since this repo is migrating to another repo, so will this repo no longer accept contributions or from now contributions are to be made in another one .

Reply:
This repo is not migrating. We're developing a new app in Flutter for both android and ios but this will stay as the native app. You can contribute to this one, the other one or even both! :)